components/QuickStartMenu.tsx

```diff
@@ -24,6 +67,62 @@ const QuickStartMenu = ({ className }: QuickStartMenuProps): JSX.Element => {
           </g>
         </svg>
       </button>
+      {/* NOTE:: Dropdown currently absolutely positioned. Revist if used in place other than the navigation */}
+      <div
+        ref={quickStartMenuRef}
+        className={classnames(
+          'shadow-sm',
+          'bg-white',
+          'border',
+          'border-gray-light',
+          'absolute',
+          'top-20',
+          'w-56',
+          'right-1',
+          'lg:transform-none',
+          'lg:transition-none',
+          'transform-gpu',
+          {
+            [quickStartMenuInactiveClasses]: !isOpen,
+            [quickStartMenuActiveClasses]: isOpen,
+          }
+        )}
+      >
+        <div className={classnames('hidden')}>{SitecoreQuickLinks.title}</div>
```

Comment:
Two things:

1. `{SitecoreQuickLinks.title}` should probably `sr-only` text inside of the `<button>` itself.
2. Remember that `hidden` is `display: none` - use the `sr-only` class when appropriate.